pymc_marketing/mmm/tvp.py

```diff
@@ -171,7 +171,7 @@ def time_varying_prior(
         hsgp_dims = (dims[1], "m")
 
     gp = pm.gp.HSGP(m=[m], L=[L], cov_func=cov_func)
-    phi, sqrt_psd = gp.prior_linearized(Xs=X[:, None] - X_mid)
+    phi, sqrt_psd = gp.prior_linearized(X=X[:, None] - X_mid)
```

Comment:
The pyproject toml suggests that we support a few versions of PyMC
https://github.com/pymc-labs/pymc-marketing/blob/9b691a9a2f957fb011228c4ca6d4637649bb94ab/pyproject.toml#L34
It seems this keyword arg has changed on PyMC>=5.16.0
[5.12.0 docs](https://www.pymc.io/projects/docs/en/v5.12.0/api/gp/generated/classmethods/pymc.gp.HSGP.prior_linearized.html)
[5.16.0 docs](https://www.pymc.io/projects/docs/en/v5.16.0/api/gp/generated/classmethods/pymc.gp.HSGP.prior_linearized.html)

In order to support this set of versions, let's use a positional argument instead of keyword argument just to be safe:
```suggestion
    phi, sqrt_psd = gp.prior_linearized(X[:, None] - X_mid)
```

Reply:
It seems like we should hit this issue though with 0.7.0 because of the new PyMC version restriction. Think this support >=5.12.0 though